src/main/resources/static/bloodDrives.html

```diff
@@ -0,0 +1,283 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>View Feedback</title>
+    <link rel="stylesheet" href="design.css" media="screen">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.10/semantic.min.css">
+</head>
+<div class="header">
+    <div class="ui green clearing inverted segment">
+        <!--See modal section of semantic, click on login to popup the username and pw prompt-->
+        <!--signup will be a button next to login that turns into a signout after a user is logged in-->
+
+        <div class="ui right floated header">
+            <button class="ui red button" id="loginButton" onclick="signInButton()">SignUp/Login</button>
+            <!--show after logged in-->
+            <a href="/logout">
+              <button class="ui red button" id="logoutButton">Logout</button>
+            </a>
+        </div>
+        <a href="/index.html">
+          <h3 class="ui left floated huge red header">
+              Red Cross Warriors
+          </h3>
+        </a>
+    </div>
+</div>
+
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.2.10/semantic.min.js"></script>
+<script src="webpageFun.js"></script>
+<nav>
+    <!--whatever pages we want-->
+    <div class="ui attached stackable menu">
+        <!--how to make this fully left l-->
+        <div class="ui container">
+            <a class="item" onclick="location.href='index.html'">
+                <i class="home icon" id="homeButton"></i> Home
+            </a>
+            <a class="item" onclick="location.href='contactus.html'">
+                <i class="mail icon"></i> Contact Us
+            </a>
+			      <a class="item" onclick="location.href='myProfile.html'">
+                 <i class="address card icon"></i>Profile
+            </a>
+            <a class="item" onclick="location.href='bloodDrives.html'">
+                <i class="tint icon"></i> Blood Drives
+            </a>
+            <!--how to lock this behind login-->
+            <div class="ui simple dropdown item" id="adminNav">
+                Admin
+                <i class="dropdown icon"></i>
+                <div class="menu">
+                  <a class="item" href="/addBloodDrive"></i>Add Blood Drive</a>
+                  <a class="item"><i class="globe icon"></i> Add News and Events</a>
+                  <a class="item"><i class="settings icon"></i> Account Settings</a>
+                </div>
```

Comment:
Would it be possible to hide the admin navigation buttons if your on logged in as an admin? 
or is this change done in your other branch?

Reply:
I do that in the fix/nav branch. In that branch the my profile page wont show unless you are signed in and i completely removed the add blood drive from nav. You would get to the add blood drive page from blood drive page itself.